libs/ui/src/reports/precinct_scanner_tally_reports.tsx

```diff
@@ -22,9 +21,8 @@ export interface PrecinctScannerTallyReportsProps {
   pollsTransition: StandardPollsTransitionType;
   isLiveMode: boolean;
   pollsTransitionedTime: number;
+  reportPrintedTime: number;
```

Comment:
Besides removing the QR code page in this commit, I parameterized the `reportPrintedTime` for testing purposes.